language/Japanese/strings.po

```diff
@@ -5312,197 +5338,197 @@ msgstr "タイプ"
 
 #: plugins/cheatfind/init.lua:411
 msgid "(empty)"
-msgstr ""
+msgstr "(空)"
 
 #: plugins/cheatfind/init.lua:427
 msgid "You can enter any type name"
-msgstr ""
+msgstr "任意のタイプ名を入力できます"
 
 #: plugins/cheatfind/init.lua:436
 msgid "Save"
 msgstr "保存"
 
 #: plugins/cheatfind/init.lua:444
 msgid "Type name is empty"
-msgstr ""
+msgstr "タイプ名が空です"
 
 #: plugins/cheatfind/init.lua:468
 #, lua-format
 msgid "Cheat written to %s and added to cheat.simple"
-msgstr "チートを%sに書き込み、cheat.simpleに追加しました"
+msgstr "チートを %s に書き込み、cheat.simple に追加しました"
 
 #: plugins/cheatfind/init.lua:478
 msgid "Cheat added to cheat.simple"
-msgstr "チートをcheat.simpleに追加しました"
+msgstr "チートを cheat.simple に追加しました"
 
 #: plugins/cheatfind/init.lua:483
 msgid ""
 "Unable to write file\n"
 "Ensure that cheatpath folder exists"
 msgstr ""
 "ファイルに書き込めません\n"
-"cheatpathフォルダが存在しているか確認してください"
+"cheat パスのフォルダが存在するか確認してください"
```

Comment:
This change is incorrect – `cheatpath` is the name of an option as it appears on the command line or in INI files.